src/components/Header/Header.component.jsx

```diff
@@ -16,14 +16,27 @@ import { SearchVideos } from '../../resources/calls';
 
 import { useAuth } from '../../providers/Auth';
 import { useSearch } from '../../hooks/SearchProvider/SearchProvider';
+import { useTheme } from '../../hooks/ThemeProvider/ThemeProvider';
 import { fetchSearch } from '../../utils/fetchApi';
 
 function Header() {
   const history = useHistory();
   const { authenticated, logout } = useAuth();
   const [openMenu, setOpenMenu] = useState(false);
+  const [darkToggle, setDarkToggle] = useState(false);
   const [term, setTerm] = useState('wizeline');
   const { changeSearch } = useSearch();
+  const { setTheme } = useTheme();
+
+  useEffect(() => {
+    if (darkToggle) {
+      setTheme({ type: 'DARKMODE' });
+      return;
+    }
+
+    setTheme({ type: 'LIGHTMODE' });
```

Comment:
this can be reduced with a ternary operator:
```js
darkToggle ? setTheme({ type: 'DARKMODE' }) : setTheme({ type: 'LIGHTMODE' });
```

Reply:
update: I thought another elegant way to express it:
```js
setTheme({ type: `${darkToggle ? 'DARKMODE' : 'LIGHTMODE'}` })
```

Reply:
Thank you